edu.stanford.protege.appdata.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
     <id>edu.stanford.protege</id>
-    <launchable type="desktop-id">edu.stanford.protege.desktop</launchable>
```

Comment:
Are you sure this is a good idea to remove this?

> Data will only be fetched from a desktop file if one <launchable/> tag is present to define a .desktop file ID. If multiple launchable tags are defined, no data will be merged in from .desktop files. 

https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html

Reply:
GitHub apparently stripped the `<launchable/>` tag from my comment.

Reply:
Yes, you duplicated it in the file as the commit message says.